sonaranalyzer-dotnet/src/SonarAnalyzer.Common/Helpers/DiagnosticDescriptorBuilder.cs

```diff
@@ -74,29 +74,7 @@ public static DiagnosticDescriptor WithSeverity(this DiagnosticDescriptor descri
                 (string)descriptor.Description,
                 descriptor.HelpLinkUri,
                 descriptor.CustomTags.ToArray());
-        }
-
-        /// <summary>
-        /// Creates a new DiagnosticDescriptor instance copying all the properties from the original, overriding the
-        /// IsEnabledByDefault.
-        /// </summary>
-        /// <param name="descriptor">DiagnosticDescriptor instance to copy.</param>
-        /// <returns>
-        /// A new DiagnosticDescriptor instance with overridden value of the IsEnabledByDefault property.
-        /// </returns>
-        public static DiagnosticDescriptor DisabledByDefault(this DiagnosticDescriptor descriptor)

```

Comment:
This is no longer required as we want all rules to be enabled by default (and disable by the new mechanism).